src/ansiblelint/yaml_utils.py

```diff
@@ -1,4 +1,5 @@
 """Utility helpers to simplify working with yaml-based data."""
+# pylint: disable=too-many-lines
```

Comment:
Uhm. I'm not sure how else to organize this better.
The YAML formatting bits (`ruamel.yaml` extensions) make up over half of this file (over 550 lines).
Then there's another large chunk of utils used to navigate through YAML files.
But, I'm not sure what to name any new modules.